### PR TITLE
Fix enforcing `chunkSize` when TUS is not enabled and update the default `TUS_CHUNK_SIZE` to `8mb`

### LINKED ISSUE #24001: Invalid chunkSize provided - breaking changes in 11.2.x when using gcs-storage-driver
### Describe the Bug

After updating from v11.1.2 to the latest v11.2.x the container keeps crashing and restarting with error `Invalid chunkSize provided` 

The endpoint `/server/health` is failing which would in docker/k8s cause problems with healthchecks and restarts.

Even with `TUS_ENABLED` set to `false`, if I don't have `TUS_CHUNK_SIZE` defined with appropriate value such as `1024kb`, it would throw this error as the default value for the `gcs` storage driver is incorrect.

Docs:
https://github.com/directus/directus/blob/main/docs/self-hosted/config-options.md

> The TUS_CHUNK_SIZE for storage-driver-gcs must be a power of 2 with a minimum of 256kb (e.g. 256kb, 512kb, 1024kb).

Default value:
https://github.com/directus/directus/blob/main/packages/env/src/constants/defaults.ts#L129

> TUS_CHUNK_SIZE: '10mb',

The default value for the `storage-driver-gcs` for this to work and closest to the `10mb` point would be `8192kb` or `16384kb`

My opinion is that if I have `TUS_ENABLED` set to false, I should not need to set `TUS_CHUNK_SIZE` value at all. 

firefox_xA9SfSCM3J

### To Reproduce

I have a deployment with k8s and of course you need gcs storage, otherwise you can have just a config with the database and the storage and thats it.

### Directus Version

v11.2.0 + v11.2.1

### Hosting Strategy

Self-Hosted (Docker Image)

### Database

Postgresql 16

### PR DESCRIPTION
**Pull request by ComfortablyCoding**
## Scope

What's changed:

- We no longer incorrectly enforce chunkSize if TUS is not enabled
- Updated the default chunkSize to `8mb`

## Potential Risks / Drawbacks

- None

## Review Notes / Questions

- None

---

Fixes #24001

### CODE REVIEW

**Review by licitdev — COMMENTED** [review]
Let's update the default chunk size to `8mb` which should be a valid chunk size across drivers.

On a side note, I can confirm that `8mb` chunk size works for the S3 driver.

**Comment by ComfortablyCoding** [comment]
> Let's update the default chunk size to `8mb` which should be a valid chunk size across drivers.

Agreed, added via 6aafc99

**Review by licitdev — APPROVED** [review]

**Review by licitdev — CHANGES_REQUESTED** [review]
Changeset needs to be updated

**Comment by ComfortablyCoding** [comment]
> Changeset needs to be updated

Updated, should we consider `env` change as major? Technically changing the default is a breaking change

File: .changeset/cyan-sloths-appear.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+'@directus/storage-driver-azure': patch
+'@directus/storage-driver-gcs': patch
+'@directus/constants': patch
+'@directus/env': major
+'docs': patch
+'@directus/api': patch
+'@directus/app': patch
+---
+
+Fixed enforcing `chunkSize` when TUS is not enabled
```
Comment (licitdev) [inline .changeset/cyan-sloths-appear.md:13]:
```suggestion
Fixed enforcing `chunkSize` when TUS is not enabled and updated the default `TUS_CHUNK_SIZE` to `8mb`
```

**Review by licitdev — APPROVED** [review]

**Pull request merged**